groups/models.py

```diff
@@ -161,6 +161,20 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def subscribers(self):
```

Comment:
This method name is a bit misleading, since a comment doesn't have subscribers.  Is this method ever going to be used outside of the view you've taken it from?